build.sbt

```diff
@@ -350,8 +350,17 @@ val mtagsSettings = List(
     if (isCI) Nil
     // NOTE(olafur) pprint is indispensable for me while developing, I can't
     // use println anymore for debugging because pprint.log is 100 times better.
-    else
-      List("com.lihaoyi" %% "pprint" % "0.6.6")
+    else {
+      val scalaV = scalaVersion.value
+      val isScala3RC = scalaV.stripPrefix("3.0.0-").startsWith("RC")
```

Comment:
Do we need it at all in `3.0.0-RC*` ? That's just a utility useful while testing, let's instead deprecate RC versions

Reply:
It's hard to say ))
Maybe let's wait for the end of exams and deprecate RC after them?
Just to avoid bothering students by `You are using legacy ... Please upgrade` notifications while they are preparing for exams.